src/js/delete_wordles.js

```diff
@@ -1,34 +1,45 @@
 "use strict";
 
-/* global globalOptions, inverted_wordles, openDialog, closeDialog */
+/* global inverted_wordles, aria */
 
-// Bind events for delete buttons
-inverted_wordles.bindDeleteEvents = function (containerElm) {
-    const delButtons = containerElm.querySelectorAll(globalOptions.selectors.deleteButton);
+/**
+ * Bind event listeners for "Delete a wordle" buttons.
+ * @param {DOMElement} containerElm - The DOM element of the holding container to find delete buttons.
+ * @param {Object} options - The value of inverted_wordles.manage.globalOptions.
+ */
+inverted_wordles.manage.bindDeleteEvents = function (containerElm, options) {
```

Comment:
Similarly with bindInputEvents, explain that the operation of this function is meant to be incremental - it will accept any a DOM container at any level of containment and bind any delete buttons that it finds in there - it is the caller's responsibility to ensure that it is called only once on each newly generated block of markup.